kamailio/db_scripts/mysql-specific

```diff
@@ -40,5 +40,6 @@ EOF
 sql_footer() {
 cat << EOF
 ALTER TABLE location_attrs MODIFY avalue varchar(512);
+ALTER TABLE presentity MODIFY etag varchar(128);
```

Comment:
why this change ?

Reply:
When in tested i can see errors that value `etag` cannot be saved as too long value.
I adjusted presentity to fix this.